thunderbird-development/codebase-overview/chat/message-styles.md

```diff
@@ -60,7 +60,7 @@ These labels will be replaced with message-specific data when used in HTML files
 ```
 
 * **time**, **time{format}**: The time when the message was sent. Takes an optional format parameter \(unfortunately, the formats supported are not the same on all the OSes as this internally calls the native strftime function of the OS\),
-* **timestamp**: The time when the message was sent, as an integer value \(number of seconds since 1970\). Useful to compute intervals between messages. Added between 0.2beta2 and 0.2 final,
+* **timestamp**: The time when the message was sent, as an integer value \(number of seconds since 1970\). Useful to compute intervals between messages. Added to Instantbird between 0.2beta2 and 0.2 final,
```

Comment:
Should we not remove the entire sentence, if those version numbers refer to Instabird and not Thunderbird?

Reply:
I opted for not removing references to Instantbird, but I'd happily do so.

Reply:
I had been removing all references to Instantbird, must have missed these!